templates/constraints/yml/templates/csharp/sso-tab/teamsapp.local.yml.tpl.mustache

```diff
@@ -12,14 +12,37 @@ provision:
 {{#aadAppUpdate}} {{/aadAppUpdate}}
 
 {{#teamsAppValidateManifest}} {{/teamsAppValidateManifest}}
+
 {{#teamsAppZipAppPackage}} {{/teamsAppZipAppPackage}}
+
 {{#teamsAppValidateAppPackage}} {{/teamsAppValidateAppPackage}}
+
 {{#teamsAppUpdate}} {{/teamsAppUpdate}}
 
+{{#teamsAppExtendToM365}} {{/teamsAppExtendToM365}}
+
 {{#fileCreateOrUpdateJsonFile}}
 {
   "launchSettings": true,
-  "applicationUrl": "\"https://localhost:44302;http://localhost:2544\"",
-  "hotReload": true
-}
+  "profiles": [
+    {
+      "profileName": "Microsoft Teams (browser):",
+      "applicationUrl": "\"https://localhost:44302;http://localhost:2544\"",
+      "launchUrl": "\"https://teams.microsoft.com/l/app/${{TEAMS_APP_ID}}?installAppPackage=true&webjoin=true&appTenantId=${{TEAMS_APP_TENANT_ID}}&login_hint=${{TEAMSFX_M365_USER_NAME}}\"",
+      "hotReload": true
+    },
+    {
+      "profileName": "Microsoft 365 app (browser):",
+      "applicationUrl": "\"https://localhost:44302;http://localhost:2544\"",
+      "launchUrl": "\"https://www.office.com/m365apps/${{M365_APP_ID}}?auth=2&login_hint=${{TEAMSFX_M365_USER_NAME}}\"",
+      "hotReload": true
+    },
+    {
+      "profileName": "Outlook (browser):",
+      "applicationUrl": "\"https://localhost:44302;http://localhost:2544\"",
+      "launchUrl": "\"https://outlook.office.com/host/${{M365_APP_ID}}?login_hint=${{TEAMSFX_M365_USER_NAME}}\"",
```

Comment:
Perhaps we should confirm with Yuan Tian if this URL is correct. It might be written like this: `https://outlook.office.com/mail?appTenantId=${{TEAMS_APP_TENANT_ID}}&login_hint=${{TEAMSFX_M365_USER_NAME}}`.

Reply:
Ok, I'll check with him.